### ISSUE TITLE
Website cannot be opened from the Facebook browser

### ISSUE THREAD

**Issue by dobromir-hristov** [pulilab/vue-people#113]
While trying to encourage people to tag them selves on the map, people coming from Facebook had this issue:

There is a security error:
There are problems with the security certificate on this site.

I saw you are using Lets Encrypt so it should be fine... Maybe something related to this?
https://community.bitnami.com/t/lets-encrypt-causing-security-warning-on-android-facebook-app/46885/2

**Comment by lordfuoco** [comment]
@dobromir-hristov Thank you for the issue ( and sorry for replying only now I somehow missed it )
This needs some investigation,  I am marking this as an issue for the hacktoberfest

**Comment by dobromir-hristov** [comment]
Yeah man no probs :) I just got some angry FB users nagging "my service" is lame and not working....
